contracts/MasterChefV2.sol

```diff
@@ -128,11 +128,11 @@ contract MasterChefV2 is BoringOwnable, BoringBatchable {
         emit LogSetPool(_pid, _allocPoint, overwrite ? _rewarder : rewarder[_pid], overwrite);
     }
 
-
-    /// @notice View function to see pending SUSHIs on frontend.
+    /// @notice View function to see pending SUSHI on frontend.
     /// @param _pid The index of the pool. See `poolInfo`.
-    /// @param _user address of user
-    function pendingSushi(uint256 _pid, address _user) external view returns (uint256) {
+    /// @param _user Address of user.
+    /// @return pending SUSHI reward for a given user.
+    function pendingSushi(uint256 _pid, address _user) external view returns (uint256 pending) {
```

Comment:
If we name the return we could also remove the explicit return, right?

Reply:
yes, good idea. That change also reduces lines of code for `_pendingSushi`  reference:

`pending = int256(user.amount.mul(accSushiPerShare) / ACC_SUSHI_PRECISION).sub(user.rewardDebt).toUInt256();`

I have added this to PR.